spacecmd/src/bin/spacecmd

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python3
```

Comment:
Hmm, it also makes sense to check with rpmlint and other tools in OBS, if I remember right there was a warning in case of using `env ...` in the shebang.

Reply:
That's probably right, I was thinking of changing the address regexp to now match `/usr/bin/env python3`. In the end, we'll still ship `spacecmd` with the appropriate shebang. We will probably revisit this when/if we switch to singlespec.

Reply:
I tested this on Mac so can't speak to the specfile.